client/consensus/Cargo.toml

```diff
@@ -3,27 +3,22 @@ name = "fc-consensus"
 version = "2.0.0-dev"
 authors = ["Parity Technologies <admin@parity.io>"]
 description = "Frontier consensus for substrate"
-edition = "2018"
+edition = "2021"
 license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 repository = "https://github.com/paritytech/frontier/"
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "2.0.0", features = ["derive"] }
-sp-core = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate" }
-sp-blockchain = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate" }
-sp-runtime = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate" }
-sp-api = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate" }
-sc-client-api = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate" }
-sp-block-builder = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate" }
-sp-inherents = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate" }
+async-trait = "0.1"
+thiserror = "1.0"
+
+sp-api = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
```suggestion
sp-api = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate" }
```

Just a nit but `branch = "master"` is redundant.

Reply:
I know it will use the default branch if ignore the branch, but I prefer to add the branch name explicitly